products/views.py

```diff
@@ -1,3 +1,63 @@
-from django.shortcuts import render
+import datetime
 
-# Create your views here.
+from django.utils         import timezone
+from django.views         import View
+from django.http.response import JsonResponse
+from django.db.models     import When, Case, Sum
+
+from .models import Category, Product, BannerImage
+
+class NavView(View):
+    def get(self, request):
+        category_names = []
+        
+        categories = Category.objects.all()
+        
+        for category in categories:
+            category_info = {
+                'category_id' : category.id,
+                'category_name' : category.name
+            }
+            category_names.append(category_info)
```

Comment:
```suggestion
        category_names = [
            {
                'category_id' : category.id,
                'category_name' : category.name
            } for category in Category.objects.all()]
```
가장 파이썬스러운 코드 리스트 컴프리헨션을 사용하면 이렇게 줄여볼 수 있습니다.

Reply:
넵! 예시 감사합니다~!!!! 👍🏻